server/events/comment_parser.go

```diff
@@ -341,20 +358,20 @@ Usage:
 
 Examples:
   # run plan in the root directory passing the -target flag to terraform
-  atlantis plan -d . -- -target=resource
+  atlantis plan -d . -- -target=resource{{if not .ApplyDisabled}}
 
   # apply all unapplied plans from this pull request
   atlantis apply
 
   # apply the plan for the root directory and staging workspace
-  atlantis apply -d . -w staging
+  atlantis apply -d . -w staging{{end}}
 
 Commands:
   plan     Runs 'terraform plan' for the changes in this pull request.
            To plan a specific project, use the -d, -w and -p flags.
-  apply    Runs 'terraform apply' on all unapplied plans from this pull request.
+{{if not .ApplyDisabled}}  apply    Runs 'terraform apply' on all unapplied plans from this pull request.
            To only apply a specific plan, use the -d, -w and -p flags.
-  unlock   Removes all atlantis locks and discards all plans for this PR.
+{{end}}  unlock   Removes all atlantis locks and discards all plans for this PR.
```

Comment:
```suggestion
{{- end}}
  unlock   Removes all atlantis locks and discards all plans for this PR.
```

Reply:
You'll have to line up the whitespace on these suggestions but you get the idea.